modules/administration/pages/ssl-certs-imported.adoc

```diff
@@ -70,6 +68,12 @@ Certificates signed with different CAs do not match.
 
 == Replace Certificates
 
+
+[IMPORTANT]
+====
+Use [literal]``mgrctl term`` before running steps inside the server container.
```

Comment:
I think the key point here is, that the customers do not know that they need to run this inside of the container.
So maybe we rephrase this a bit:

>   The following procedure need to run inside of the container. Use `mgrctl term` to open a terminal in the container before you execute it.

or something like this.